lib/net/peer.js

```diff
@@ -1236,8 +1236,8 @@ class Peer extends EventEmitter {
       return invTypes.FILTERED_BLOCK;
 
     if (this.options.compact
-        && this.hasCompactSupport()
-        && this.hasCompact()) {
+      && this.hasCompactSupport()
+      && this.hasCompact()) {
```

Comment:
i dont think this linting is necessary but ok ... automatic from your IDE perhaps?